src/Sylius/Bundle/CoreBundle/Installer/Executor/CommandExecutor.php

```diff
@@ -74,4 +99,16 @@ private function getDefaultParameters(): array
 
         return $defaultParameters;
     }
+
+    protected function createProgressBar(OutputInterface $output, int $length = 10): ProgressBar
```

Comment:
this class is final, so eather it should be private or it should be removed if we choose to remove the runCommands method.